CONTRIBUTING.md

```diff
@@ -1,11 +1,14 @@
- 
+
 # Contribution Guide
 Below are the instructions for making changes to simple-chat. There are a few important guidelines we'd wish for you to follow.
 These will make it easier for you to contribute and for us to follow. Thank you for contributing!
 
 ## Getting Started
 If you've found a bug and don't have a solution or have a feature request, please [create an issue](https://github.com/mbrandau/simple-chat/issues/new).
-If you found a bug and have a solution, please follow these steps:
+
+Ensure the bug was not already reported by searching on GitHub under [Issues](https://github.com/mbrandau/simple-chat/issues).
+
+If you're unable to find an open issue addressing the problem, open a new one. Be sure to include a title and clear description, as much relevant information as possible and follow these steps:
```

Comment:
After creating an issue you don't necessarily have to fork the repo and make changes. You should add something like
`and follow these steps if you already have a solution:`